beacon-chain/rpc/eth/builder/handlers_test.go

```diff
@@ -38,46 +37,46 @@ func TestExpectedWithdrawals_BadRequest(t *testing.T) {
 	testCases := []struct {
 		name         string
 		path         string
-		urlParams    map[string]string
+		urlParams    []string
```

Comment:
why is this an array of string now, we should keep the map... or make it a function where the request is passed in to set

Reply:
or at least some other way of doing this, like param name, param value, this seems like it's prone to bad coding practices